xesmf/util.py

```diff
@@ -96,3 +150,197 @@ def grid_global(d_lon, d_lat):
                       'might not cover the globe uniformally'.format(d_lat))
 
     return grid_2d(-180, 180, d_lon, -90, 90, d_lat)
+
+
+def is_longitude(var):
+    """Return True if variable is a longitude.
+
+    A variable is considered a longitude if its `units` are 'degrees_east' or a synonym.
+
+    Parameters
+    ----------
+    var: xr.Variable
+      Coordinate variable.
+
+    Return
+    ------
+    bool
+      True if variable is a longitude.
+
+    Example
+    -------
+    >>> # Identify the longitude coordinate in a dataset.
+    >>> lon = next(filter(is_longitude, ds.coords))
+    """
+    return has_units_of_longitude(var)
+
+
+def is_latitude(var):
+    """Return True if variable is a latitude.
+
+    A variable is considered a latitude if its `units` are 'degrees_north' or a synonym.
+
+    Parameters
+    ----------
+    var: xr.Variable
+      Coordinate variable.
+
+    Return
+    ------
+    bool
+      True if variable is a latitude.
+    """
+    return has_units_of_latitude(var)
+
+
+def has_name_of_longitude(var):
+    key = "standard_name"
+    return var.attrs.get(key) == "longitude"
+
+
+def has_name_of_latitude(var):
```

Comment:
A minimal doc? Well, the name is explicit. Perhaps just to say that the input is a datarray.

Reply:
Done